navigation/presentation.html

```diff
@@ -2,43 +2,1222 @@
+                  </li>
+                </ul>
+              </div>
+            </div>
+
+            <aside class="notes">
+              3D Touch is a feature, which triggers a small submenu you can use to run specific commands.
+              All you have to do is to press hard on an app icon.
+              <br /><br />
+              It was first introduced by Apple. Later, a similar feature appeared on android called "App Shortcuts" and allows you to have an experience similar to "3D Touch".
+              <br /><br />
+              It’s a way to create a navigation shortcut, showing the key actions for the selected app.
+              Another common use for this mobile navigation is the preview of content.
+              <br /><br />
+              Some popular apps that support 3D Touch include Amazon, Chrome, Gmail, Google Maps, Instagram, Netflix, Twitter, Uber, WhatsApp, YouTube, etc.
+            </aside>
+          </section>
+        </section>
+        <!-- 4 slide -->
+        <section class="slide-with-text-and-logos">
+          <h4 class="slide-name">Navigation libraries</h4>
+          <div>
+            <div>
+              <p><b>React Navigation: </b>well documented, actual, expo, community, JS</p>
+              <p><b>React Native Navigation: </b>native, actual, has good documentation, doesn’t work with expo</p>
+              <p><b>React-native-router-flux: </b>based on React Navigation</p>
+              <p><b>React Router Native: </b>easy to use if familiar with ReactJS and react-router-dom</p>
+            </div>
+            <div class="logos">
+              <img src="assets/images/logo_react_native_navigation.png" alt="logo" />
+              <img src="assets/images/logo_react_navigation.svg" alt="logo" />
+              <img src="assets/images/logo_react_router_native.svg" alt="logo" />
+            </div>
+          </div>
+
+          <aside class="notes">
+            In React Native, we have the option of setting up navigation by creating our own native modules or by building our own JavaScript-based implementation.
+            There are also a number of community-supported libraries that we can use for either of these approaches.
+            <br /><br />
+            Some of them you can see on the slide.
+            <br />
+            - <strong>React navigation </strong>is widely used, well documented. It delivers out-of-the-box, pre-styling navigators such as Stack Navigator, Bottom Tab… You can also customize or extend any part of React Navigation to fit your needs.
+            <br />
+            - <strong>React Native Navigation </strong>provided by Wix engineering. This is a popular alternative to React Navigation and may be a better fit for you if you are trying to integrate React Native into an existing large native app. This library doesn’t work with Expo (expo is a framework to build React Native apps, it provides you with a list of tools that simplify the creation and testing of RN app).
```

Comment:
1. Expo is not a framework it's a platform. From Github: "Expo is an open-source platform for making universal native apps that run on Android, iOS, and the web."
2. Maybe it could be better to place this information below the list of libraries on the slide like "* Expo is a platform to build React Native apps" For me it will look less confusing.

Reply:
done